src/gui/sdlmain.cpp

```diff
@@ -2876,6 +2881,11 @@ void Config_Add_SDL() {
 	Pint = Pmulti->GetSection()->Add_int("ysens",Property::Changeable::Always,100);
 	Pint->SetMinMax(-1000,1000);
 
+	pbool = sdl_sec->Add_bool("raw_mouse_input", on_start, false);
+	pbool->Set_help("Enable this setting to bypass mouse sensitivity and\n"
+	                "acceleration profile used by your operating system. This\n"
+	                "works in fullscreen and in window with captured cursor.");
```

Comment:
*"... works in fullscreen or when the cursor is captured."* I think that reads a bit better?